### DISCUSSION TITLE
New version is available message

### DISCUSSION THREAD

**Discussion by Simleybob** [magknight/787aviatorsTracker#659] [Ideas]
Just to make sure it is not forgotten:

When a stable update is available, a window should come up saying update is available via the skunkcrafts plugin! - this will get people to update and have less issues being reported here and so people will have the latest stuff

**Comment by aeroniemi** [comment]
currently i see it as:

- know what versions are available on the remote
- know whether a newer version is available than the one i have
- know whether a newer beta version is available than what i have

is there anything else that'd be useful?

**Reply by Simleybob** [reply]
Actually, on the pop up or however you are going to do it, maybe put the link to the web doc on how to update if people are new or are confused. Otherwise we’ll immediately have loads in the discord saying ‘how do you update, I tried reinstalling from the store and it didn’t work’ etc.

**Reply by aeroniemi** [reply]
The actions workflow now uploads a file containing details about all of the available versions, so now it needs to be added to the actual aircraft, with:

- Ability to select the branch you want notifications about
- Message, maybe EFB or pop-up
- Button to close x-plane to allow you to update

**Comment by Simleybob** [comment]
And of course, you have to check a box or something similar, to check for beta updates - quite a few will just want stable and don’t really care about beta or alpha.
